index.js

```diff
@@ -0,0 +1,17 @@
+const userLogin = document.querySelector("#user-login");
```

Comment:
Why leave the const in global space?

Why not:

```js
function setNameIfFound(name){
  const userLogin = document.querySelector("#user-login");
  if (name) {
    userLogin.innerHTML = `Hello, ${name}!`;    
  }
};
```

Reply:
Makes sense.